gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "2.0.6",
+  "version": "2.0.7",
```

Comment:
Should be a minor version since it's a new feature (new prop)

Reply:
> Should be a minor version since it's a new feature (new prop)

https://github.com/sonatype/sonatype-react-shared-components/pull/166/commits/c0821eb0c5fdf0162233b0efb0efad771c7f8427